support-frontend/test/selenium/util/PostDeployTestUser.scala

```diff
@@ -24,3 +28,31 @@ class PostDeployTestUser(driverConfig: DriverConfig) extends TestUser {
   val username = testUsers.generate()
   addTestUserCookies(username)
 }
+
+class PostDeployTestUserSubs(driverConfig: DriverConfig) extends TestUser {
+
+  private val client = new OkHttpClient()
+  private val requestToIdapi = new Request.Builder()
+    .url(Config.idapiNewTestUserUrl)
+    .addHeader(Config.idapiClientAccessTokenName, Config.idapiClientAccessTokenSecret)
+    .post(RequestBody.create(MediaType.parse("application/json"), "{}"))
+    .build()
+  private val response = client.newCall(requestToIdapi).execute()
+  private val responseBodyAsJson = parse(response.body().string()).getOrElse(Json.Null)
+  private val cursor = responseBodyAsJson.hcursor
+
+  private def addTestUserCookies(testUsername: String) = {
+    driverConfig.addCookie(name = "pre-signin-test-user", value = testUsername)
```

Comment:
do we need this one for subs? I think only contributions allows non signed in test users

Reply:
I am not sure about the purpose of this cookie, but would be happy to investigate further. Removing it causes some of the Stripe tests to fail, so I'd leave it in to limit the scope of the current PR.